tests/debug_tests.cpp

```diff
@@ -0,0 +1,12 @@
+// Select the test case by choosing the header and the test file
+#include "index_option.h"
+#define TESTFILE "../example_data/data/index_option.raw"
+
+int main()
+{
+    std::string file_path = TESTFILE;
+    void * builder = load(file_path);
+    std::cout << "Execution completed" << std::endl;
+
+    return 0;
+}
```

Comment:
here is the simple C++ test case, by changing the first 2 lines, we can select any test case.

I also get this error at the end of execution:

    free(): double free detected in tcache 2

the debugger stops at `struct pair` inside `stl_pair.h`, not sure what this means.